dist/twine.js

```diff
@@ -499,7 +499,8 @@ for (j = 0, len = ref.length; j < len; j++) {
 setupAttributeBinding('innerHTML', 'unsafe-html');
 
 preventDefaultForEvent = function(event) {
-  return (event.type === 'submit' || event.currentTarget.nodeName.toLowerCase() === 'a') && Twine.getAttribute(event.currentTarget, 'allow-default') !== '1';
+  var ref1;
```

Comment:
Might as well make this `ref1 = Twine.getAttribute(event.currentTarget, 'allow-default')`, right? Saves a bit of length on the next line, and the confusion of making an assignment inside a conditional.